var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -53,8 +53,10 @@ class Kokkos(CMakePackage, CudaPackage, ROCmPackage):
                                      'Aggressively vectorize loops'],
         'compiler_warnings': [False,
                               'Print all compiler warnings'],
-        'cuda_lambda': [False,
+        'cuda_constexpr': [False,
                         'Activate experimental lambda features'],
+        'cuda_lambda': [False,
+                        'Activate experimental constexpr features'],
```

Comment:
The option descriptions for the updates items looks reversed.

Reply:
> The option descriptions for the updated items looks reversed.

Good catch, fixed!

> Also, please replace "jjwilke" in the maintainers array with "DavidPoliakoff". Jeremy is no longer a maintainer.

Done.